airflow/jobs/scheduler_job.py

```diff
@@ -774,25 +775,46 @@ def _execute(self) -> None:
     @provide_session
     def _update_dag_run_state_for_paused_dags(self, session: Session = NEW_SESSION) -> None:
         try:
-            paused_dag_ids = DagModel.get_all_paused_dag_ids()
```

Comment:
Nothing is using `get_all_paused_dag_ids` any longer, "internal" and we should toss it, or "public" and need to keep it around?